inference-engine/src/gna_plugin/optimizer/gna_pass_manager.cpp

```diff
@@ -2164,14 +2185,16 @@ void MoveFakeQuantizeLayerIntoQuantParamsPass :: run() {
         // and propagate quantization data
         for (size_t i = 0; i < nextLayers.size(); ++i) {
             auto insDatas = CNNLayerFindInsDataIdxes(fqLayer->outData.front(), nextLayers[i]);
```

Comment:
I suggest to move lines 2187-2191 to if-block isFQFuseAllowed, because insDatas is not used anywhere else.

Reply:
moved